README.md

```diff
@@ -621,6 +672,36 @@ api.PathPart = "users/abc";
 await api.GetAsync();
 ```
 
+#### Serialization of Path Properties
```

Comment:
Nit: title should match TOC

Reply:
I can't see any difference?

TOC: `3. [Serialization of Path Properties](#serialization-of-path-properties)`
Title: `#### Serialization of Path Properties`

Reply:
I must have gotten confused, my bad